testing/log/service_test.go

```diff
@@ -67,7 +70,7 @@ func TestLogValidatorService_NewRequest(t *testing.T) {
 		var logName = fmt.Sprintf("test%v.log", i)
 		var fullLogname = path.Join(tempPath, logName)
 
-		toolbox.RemoveFileIfExist(fullLogname)
+		_ = toolbox.RemoveFileIfExist(fullLogname)
```

Comment:
what is `toolbox.RemoveFileIfExist(fullLogname)` return? why you use blank identifier here ?

Reply:
since this is test, file may or may not exists, thus to prepare test you remove them just in case, and ignore error in case there are no files to remove, 
blank identifies is to notify errcheck tools that we ignoring error on purpose